### PR TITLE
[ko] fix an invalid url

### PR DESCRIPTION
**Pull request by npu21**
https://github.com/kubernetes/website/pull/26400

### CODE REVIEW

File: content/ko/docs/concepts/workloads/pods/ephemeral-containers.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ API에서 특별한 `ephemeralcontainers` 핸들러를 사용해서 만들어지
 
 임시 컨테이너를 사용해서 문제를 해결하는 예시는
 [임시 디버깅 컨테이너로 디버깅하기]
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/pods/ephemeral-containers.md:78]:
```suggestion
[임시 디버깅 컨테이너로
```

File: content/ko/docs/concepts/workloads/pods/ephemeral-containers.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ API에서 특별한 `ephemeralcontainers` 핸들러를 사용해서 만들어지
...
-(/docs/tasks/debug-application-cluster/debug-running-pod/#debugging-with-ephemeral-debug-container)를 참조한다.
+(/docs/tasks/debug-application-cluster/debug-running-pod/#ephemeral-container)를 참조한다.
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/pods/ephemeral-containers.md:79]:
```suggestion
디버깅하기](/docs/tasks/debug-application-cluster/debug-running-pod/#ephemeral-container)를 참조한다.
```

**Comment by yoonian** [comment]
@npu21

Thank you.
Could you modify this also? If not it's okay.

/lgtm

**Comment by seokho-son** [comment]
@npu21 Thank you for the contribution!
Please squash  the commits according to the following guideline.
https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Pull request merged**